Dockerfile

```diff
@@ -17,15 +17,26 @@ RUN apk --no-cache add \
     texmf-dist-latexextra \
     texmf-dist-pictures \
     texmf-dist-science \
-    wget
+    wget \
+    curl \
+    git
 
 
 RUN luaotfload-tool --update
 RUN apk --no-cache add py3-pandas
-RUN apk --no-cache add pipx
-RUN pipx install cgt-calc
-RUN pipx ensurepath
 
-WORKDIR /data
+WORKDIR /build
+
+RUN curl -sSL https://install.python-poetry.org | python3 -
+RUN ln -s /root/.local/share/pypoetry/venv/bin/poetry /bin/
+COPY cgt_calc /build/cgt_calc
+COPY .git /build/.git
+COPY .github /build/.github
+COPY pyproject.toml README.md .editorconfig .gitignore .kodiak.toml .pre-commit-config.yaml /build/
```

Comment:
Why don't copy everything or clone from github? Or mount this folder inside docker, so changes inside the container are available in the outside repo.

Reply:
> Why don't copy everything?

Yes, let's do that.

> or clone from github?

because You want to build from your changes.

> Or mount this folder inside docker, so changes inside the container are available in the outside

This is kind of the point - that I don't want all the changes to be outside of the container =) If it was inside a corporate code - makes sense, but for a public repository I would vote for more isolation.